internal/cypress/docker/runner.go

```diff
@@ -3,24 +3,30 @@ package docker
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
-	"github.com/rs/zerolog/log"
-	"github.com/saucelabs/saucectl/cli/command"
-	"github.com/saucelabs/saucectl/cli/progress"
-	"github.com/saucelabs/saucectl/cli/runner"
-	"github.com/saucelabs/saucectl/cli/streams"
-	"github.com/saucelabs/saucectl/internal/cypress"
 	"io"
 	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/saucelabs/saucectl/cli/progress"
+	"github.com/saucelabs/saucectl/cli/runner"
+	"github.com/saucelabs/saucectl/cli/streams"
+	"github.com/saucelabs/saucectl/internal/cypress"
 )
 
 // SauceRunnerConfigFile represents the filename for the sauce runner configuration.
 const SauceRunnerConfigFile = "sauce-runner.json"
 
+// ErrEmptyImageProperty represents error regarding empty image docker image name property in the config file.
```

Comment:
I don't think this knowledge really belongs into the docker section.
Technically speaking, the docker section has no knowledge of any saucectl config files. All it cares about is that the the docker image is specified.

I'd just return `errors.New("no docker image specified")`, without even having the need to define a specific error type.

Reply:
I agree with you.